downstream/modules/platform/proc-controller-create-organization.adoc

```diff
@@ -12,14 +12,14 @@
 .Procedure
 . From the navigation panel, select {MenuAMOrganizations}.
 . Click btn:[Create organization].
-. Enter the *Name* and optionally provide a *Description* for your organization.
+. Enter the *Name* and provide a *Description* for your organization.
 +
 [NOTE]
 ====
 If {ControllerName} is enabled on the platform, continue with Step 4. Otherwise, proceed to Step 6.
 ====
 +
-. Select the name of the *Execution environment* or search for one that exists that members of this team can run automation.
+. Select the name of the *Execution environment* or search for one that exists that members of this organization can use to run automation.
```

Comment:
Suggestion: remove the phrase 'that exists' to say, "Select the name of the *Execution environment* or search for one that members of this organization can use to run automation."

If a user can search for something, it's implied it already exists.